### DISCUSSION TITLE
Games with custom touch controls

### DISCUSSION THREAD

**Discussion by redphx** [redphx/better-xcloud#241] [Custom touch controls]
> [!TIP]
> 🤓 [How to create custom touch control for Better xCloud](https://github.com/redphx/better-xcloud/wiki/How-to-create-custom-touch-control-layout)

This topic will be used to announce the list of games that support custom layouts.

ℹ️ If you want to suggest some changes to the layout, reply directly to it. Do not create a new comment (or it will be deleted).

Click on the "Subscribe" button on the right to get notifications when a new game is added.

<img width="260" alt="image" src="https://github.com/redphx/better-xcloud/assets/96280/4326974c-ad20-4889-b60b-7af7078b18f6">

You don't have to update Better xCloud to have access to new custom layouts.  

[How to request a custom touch controller layout](https://github.com/redphx/better-xcloud/discussions/239)

**Comment by redphx** [comment]
Palworld (with gyro support)

![Palworld](https://github.com/redphx/better-xcloud/assets/96280/03613728-0912-42d6-939d-a672dd672853)

**Reply by redphx** [reply]
Feb 01: only activate gyroscope when aiming/attacking

**Comment by redphx** [comment]
Brotato (with multiple layouts)

![Brotato](https://github.com/redphx/better-xcloud/assets/96280/2553b1a4-63aa-4a19-96a3-db14c7fc5f02)

**Comment by redphx** [comment]
RESIDENT EVIL 2 (with gyro support)

![RE 2](https://github.com/redphx/better-xcloud/assets/96280/d5b1eac7-8010-40c7-a2f0-90f1af544e48)

**Reply by Edu456789** [reply]
Se podria agresar al resident evil 3

**Reply by AhnJaeHoon-0512** [reply]
Can you apply the same touch controller to Resident evil re:3? It's almost the same control method, but if you can move the RB button, which is the avoidance button, to the right, I would appreciate it if you could move it!

**Reply by gallowscalibrator-creator** [reply]
Can you add this custom layout for RE 3 Némesis?

**Reply by Kode-Z** [reply]
I was going to create a layout this week for RE3!

**Reply by Kode-Z** [reply]
> Can you add this custom layout for RE 3 Némesis?

I’ve just submitted touch controls for RE3!

**Comment by redphx** [comment]
A Short Hike

![A Short Hike](https://github.com/redphx/better-xcloud/assets/96280/ee8d349e-3069-4314-a745-64b389d24b8b)

**Comment by redphx** [comment]
Chicory

![Chicory](https://github.com/redphx/better-xcloud/assets/96280/7b73b831-1540-4bff-8464-0cba8dd936a1)

**Comment by redphx** [comment]
Cocoon

![Cocoon](https://github.com/redphx/better-xcloud/assets/96280/e13ca89c-dccc-4e0c-9cac-b4296f04990c)

**Comment by redphx** [comment]
Forza Horizon 4  
Forza Horizon 5  
Forza Motorsports

![FH5](https://github.com/redphx/better-xcloud/assets/96280/f89a6516-18c3-4fba-874e-3126ab6ac750)

![FM](https://github.com/redphx/better-xcloud/assets/96280/6cdcf8fe-304c-4994-885d-c317a3156696)

**Comment by redphx** [comment]
FIFA 23

![FIFA 23](https://github.com/redphx/better-xcloud/assets/96280/54622248-e980-4cdc-9f97-d5367482df87)

**Comment by redphx** [comment]
Go Mecha Ball

![Go Mecha Ball](https://github.com/redphx/better-xcloud/assets/96280/9737b21b-f489-4643-a84c-1451210ff44b)

**Comment by redphx** [comment]
Turnip Boy Robs a Bank  

![Turnip Bank](https://github.com/redphx/better-xcloud/assets/96280/8e086cc6-327d-4fab-b7bb-1837a48d72ad)

**Comment by redphx** [comment]
Rise of the Tomb Raider

![Rise of the Tomb Raider](https://github.com/redphx/better-xcloud/assets/96280/df269d08-ed29-4d63-b7a3-d9d3a8565854)

**Reply by StevenMXZ** [reply]
Could you pass the same touch from Rise of Tomb Raider to Shadow of Tomb Raider? The input is the same

**Reply by redphx** [reply]
@StevenMXZ added

**Reply by AhnJaeHoon-0512** [reply]
@redphx I checked that the touch controller was added to the Shadow of Tomb raider. But the game list says that the touch controller is not supported, so can you add a support mark?

**Reply by StevenMXZ** [reply]
when you enter the game the custom touch will appear

**Comment by Tak-attack** [comment]
Shredders
![shredders_bx](https://github.com/redphx/better-xcloud/assets/149154024/d5bcddaf-0423-49c9-87e8-67745da7eb38)

**Comment by Tak-attack** [comment]
Halo Infinite (with gyro support, and Right stick / Relative mouse aiming mode)
![halo-infinite_bx](https://github.com/redphx/better-xcloud/assets/149154024/7420bc5a-4bf5-476a-8be8-b0321ea15d4b)

**Comment by redphx** [comment]
Perfeck Dark 

![perfect-dark](https://github.com/redphx/better-xcloud/assets/96280/e6608177-dc46-452c-9700-81f1330c067d)

**Comment by redphx** [comment]
DOOM (1993)

![doom-1993](https://github.com/redphx/better-xcloud/assets/96280/bc2ce306-9625-4032-bc5b-37da8d956dbc)

**Comment by Tak-attack** [comment]
Roboquest (with gylo support)
![IMG_2550](https://github.com/redphx/better-xcloud/assets/149154024/fdc17508-2c4e-47cc-b12d-79370d287c94)

**Comment by redphx** [comment]
PlateUp!

![image](https://github.com/redphx/better-xcloud/assets/96280/cafe912f-d4ae-4748-99c8-358b230a35c6)

**Comment by redphx** [comment]
Red Dead Redemption 2  (Remote Play)

![image](https://github.com/redphx/better-xcloud/assets/96280/7a56262f-07f8-497d-a248-627bdfdf5412)

**Comment by redphx** [comment]
The Quarry  
![telegram-cloud-photo-size-5-6332206246433765793-y](https://github.com/redphx/better-xcloud/assets/96280/303ff220-f13f-42dc-8ed9-dfc085411df4)

**Comment by evanpaul14** [comment]
Cyberpunk 2077 (remote play)
![Screenshot 2024-03-27 at 6 42 20 PM](https://github.com/redphx/better-xcloud/assets/120766197/d04bfc67-7dfd-4fc9-aeda-edc86d2330c3)

**Comment by Kode-Z** [comment]
DOOM II
![IMG_9898](https://github.com/redphx/better-xcloud/assets/162533196/5633a221-1271-4a4f-9d6e-177d5cdee91d)

**Comment by Kode-Z** [comment]
DOOM 64
![IMG_9895](https://github.com/redphx/better-xcloud/assets/162533196/cc0c2d76-06b2-4dd3-8b84-8bde65273dec)

**Comment by Kode-Z** [comment]
Perfect Dark Zero
![IMG_9896](https://github.com/redphx/better-xcloud/assets/162533196/b924efb4-4b65-4aa9-b2f2-c4e467037175)

**Comment by Kode-Z** [comment]
LIMBO
![IMG_9899](https://github.com/redphx/better-xcloud/assets/162533196/943eca45-fd00-4104-83f4-2c4ccd47be5f)

**Comment by Kode-Z** [comment]
INSIDE
![IMG_9900](https://github.com/redphx/better-xcloud/assets/162533196/1ce90a93-3112-42e5-9e95-faa20ff144ae)

**Comment by Kode-Z** [comment]
Arcade Paradise
![IMG_9894](https://github.com/redphx/better-xcloud/assets/162533196/62e0b8c1-b84a-416e-bc29-fe44db978bea)

**Comment by Kode-Z** [comment]
Close to the Sun
![IMG_9905](https://github.com/redphx/better-xcloud/assets/162533196/815aa811-b252-496b-8012-170635994882)

**Comment by redphx** [comment]
Harold Halibut

![Screenshot_20240417-111905](https://github.com/redphx/better-xcloud/assets/96280/07e6af76-e75e-4cf6-80ae-8330a98e9df4)

**Comment by Kode-Z** [comment]
Those Who Remain
![IMG_0114](https://github.com/redphx/better-xcloud/assets/162533196/3e70643a-49de-4fbe-b936-5637dff0e3c6)

**Comment by redphx** [comment]
Balatro

![screen](https://github.com/user-attachments/assets/a4720912-934f-4fc3-9d3c-a90a04d80f4e)

**Comment by Kode-Z** [comment]
South of Midnight
![image](https://github.com/user-attachments/assets/730ae0b5-47dd-4346-b463-b9ccf0d4f5a9)

**Comment by Kode-Z** [comment]
Resident Evil (Remote Play)
_Must use the “Alternate” control setting in game!_

Tap the “+” in top right corner to toggle between modern analog/D Pad tank controls.

![IMG_3678](https://github.com/user-attachments/assets/af3be472-74a8-47bb-908a-591e7f8c6626)
![IMG_3679](https://github.com/user-attachments/assets/fe7a7ce6-6b98-491c-8931-f410393a23a6)

**Reply by RC1992Jules** [reply]
Awesome! I'm definitely going to use this because this game is on my backlog and playing on the go without a controller will make it more accessible. I hope I don't sound ungrateful, but any chance of you adding Custom Touch Controls for Resident Evil 0 since they have a similar control set up?

**Reply by Kode-Z** [reply]
Cheers @RC1992Jules! It's on my todo list! 👍🏻

**Comment by Kode-Z** [comment]
Conker: Live & Reloaded (Remote Play)

![image](https://github.com/user-attachments/assets/7f89e4f8-0b55-494b-8ee0-e22c26a8bf21)

**Comment by Kode-Z** [comment]
DOOM & DOOM II

![image](https://github.com/user-attachments/assets/8e1f9bcb-bd94-4bc9-af22-0c03b9531824)

**Comment by Kode-Z** [comment]
Resident Evil 7 (Remote Play)

<img width="2796" height="1290" alt="image" src="https://github.com/user-attachments/assets/936acfb1-9f66-4f2e-babf-0cce862fed58" />

**Comment by Kode-Z** [comment]
Resident Evil Village (Remote Play)

<img width="2796" height="1290" alt="image" src="https://github.com/user-attachments/assets/9b9a3dc2-b001-47b5-b45b-bbf887e3c8ab" />

**Comment by Kode-Z** [comment]
RoboCop: Rogue City / RoboCop: Rogue City - Unfinished Business
_Includes gyro support!_

<img width="2796" height="1290" alt="image" src="https://github.com/user-attachments/assets/636074fe-f4b8-4b26-8e68-3d6cfa5e95ba" />

**Comment by Kode-Z** [comment]
Choo Choo Charles

![IMG_4376](https://github.com/user-attachments/assets/0674e519-88f4-456f-aebf-27d895f69da2)

**Comment by Kode-Z** [comment]
High on Life

<img width="2796" height="1290" alt="IMG_4413" src="https://github.com/user-attachments/assets/5e71cd1d-b076-47fe-9c0b-21d66e46c2c7" />

**Comment by Kode-Z** [comment]
Grand Theft Auto 4 (also includes the standalone Episodes from Liberty City) (Remote Play)

<img width="2796" height="1290" alt="IMG_4340" src="https://github.com/user-attachments/assets/2d60fa18-3d4b-41de-b815-169e7245e0a0" />
<img width="2796" height="1290" alt="IMG_4424" src="https://github.com/user-attachments/assets/cd92a629-a99d-4ae1-accb-a69a2ae8cc42" />

**Comment by Kode-Z** [comment]
Grand Theft Auto V (Both Xbox One/Xbox Series versions)
<img width="2796" height="1290" alt="IMG_4450" src="https://github.com/user-attachments/assets/4fa46e83-ba8a-441e-b721-e153833ec8ac" />

**Comment by Kode-Z** [comment]
Grand Theft Auto: The Definitive Edition

Grand Theft Auto 3: Definitive Edition
<img width="2796" height="1290" alt="IMG_4435" src="https://github.com/user-attachments/assets/adde5ff8-9ce2-4565-991c-dc57618b8866" />

Grand Theft Auto - Vice City: Definitive Edition
<img width="2796" height="1290" alt="IMG_4451" src="https://github.com/user-attachments/assets/32d38991-5c17-431d-ad40-10e2ba1d2ba3" />

Grand Theft Auto - San Andreas: Definitive Edition
<img width="2796" height="1290" alt="IMG_4434" src="https://github.com/user-attachments/assets/4905e84c-77a7-4e9d-993a-ba4c9a014de4" />

**Comment by Kode-Z** [comment]
Resident Evil 4 Remake (Remote Play):
_Select default control layout Type A-1!_
<img width="2796" height="1290" alt="IMG_4611" src="https://github.com/user-attachments/assets/12116c1b-e1ad-4ea9-8037-6899b995457e" />

**Comment by Kode-Z** [comment]
Quake (Xbox Cloud/Remote Play):
<img width="2796" height="1290" alt="IMG_4601" src="https://github.com/user-attachments/assets/7ba28d8a-7db3-4b0f-a76c-b2d02d1cc50b" />

_For optimal gameplay experience with this custom layout, ensure that “Always Run” is toggled “OFF” in the Options menu as the left joystick activates running when it goes past the circle threshold!_
![SIX_82BB1E17-743C-45F2-A2E5-5098CE599848](https://github.com/user-attachments/assets/10b31deb-5c69-4dc7-8215-cfe9dfaa332f)

**Comment by Kode-Z** [comment]
Sonic Mania (Remote Play):
<img width="2796" height="1290" alt="IMG_4610" src="https://github.com/user-attachments/assets/3e12f52d-cb3a-447d-be21-1d90d5cf7616" />

**Comment by Kode-Z** [comment]
Gears of War: Reloaded (Xbox Cloud/RemotePlay)
<img width="2796" height="1290" alt="IMG_4649" src="https://github.com/user-attachments/assets/9deba553-f6fa-4120-ae97-53d7066b703d" />

**Comment by Kode-Z** [comment]
Metal Gear Solid: HD Collection Xbox 360 (MGS2 & MGS3) (Remote Play):
![image](https://github.com/user-attachments/assets/da7165cf-e115-4cb4-9e0d-291661e59c97)

![image](https://github.com/user-attachments/assets/3023bbc2-f4c7-4e1b-b236-47831ab63d7e)

_Choose between control layouts for the correct game you are playing via the Better xCloud menu!_
![image](https://github.com/user-attachments/assets/f17b8e67-4c0a-4cdd-8c3b-96c8f7d9fd74)

**Comment by Kode-Z** [comment]
Metal Gear Solid 2 (Master Collection Vol. 1 Version) (Remote Play):
![image](https://github.com/user-attachments/assets/3eb0c07e-6fb9-4ae1-94b8-a6e9d0f383a8)

**Comment by Kode-Z** [comment]
Metal Gear Solid 3 (Master Collection Vol. 1 Version) (Remote Play):
![image](https://github.com/user-attachments/assets/1ba5ae0b-3fe8-4045-b7ea-3ef581cc2352)

**Comment by Kode-Z** [comment]
Metal Gear Solid 1 (Master Collection Vol. 1 Version) (Remote Play):
![image](https://github.com/user-attachments/assets/05f93f20-4002-4361-bf15-5c14a3b0c1af)

**Comment by Kode-Z** [comment]
Shenmue 1 (Remote Play):
_Includes a toggle to switch between joystick/dpad on the top right!_

![image](https://github.com/user-attachments/assets/447665c3-ee0b-44f3-a630-eca65869880b)

![image](https://github.com/user-attachments/assets/ea21ae97-4908-4d50-8339-a4fe00f685d7)

**Comment by Kode-Z** [comment]
Shenmue 2 (Remote Play):
_Includes a toggle to switch between joystick/dpad on the top right!_

![image](https://github.com/user-attachments/assets/233afb7d-730d-4cad-be36-1d993b2eeade)

![image](https://github.com/user-attachments/assets/244861f7-7649-47fb-bf48-a3c04d3e87fb)

**Comment by Kode-Z** [comment]
Hogwarts Legacy (Xbox Cloud/Remote Play):
![IMG_5273](https://github.com/user-attachments/assets/7f382e85-0465-48fa-9d08-a11744b08930)
<img width="1335" height="616" alt="IMG_5274" src="https://github.com/user-attachments/assets/3b9bef5d-1a0d-4c33-9a49-5c74ce89eadd" />

The right trigger button has segmented pull functionality.

**Comment by lowericarlos-bit** [comment]
Can you create custom buttons for Fortnite Save the World, please? It's very difficult with the xCloud buttons, or it needs to be more customizable when you change the size and position of the controls.
<img width="1600" height="720" alt="Screenshot_20251109-231700_Better xCloud" src="https://github.com/user-attachments/assets/a41a31fd-448f-468f-8937-16e333f361d1" />

**Reply by Coolkid120566** [reply]
Let me see what I can do

**Comment by Kode-Z** [comment]
Wolfenstein 3D (Remote Play):
![IMG_5349](https://github.com/user-attachments/assets/314fcfc7-f4b1-4f4d-9ba7-b2e6011d5651)

**Reply by emmanuelekedlmma-maker** [reply]
Hi

**Comment by Kode-Z** [comment]
Alan Wake 2 (Remote Play):
![IMG_5490](https://github.com/user-attachments/assets/0767d248-39d7-491f-a7cb-bf341f8cff8d)

**Comment by Kode-Z** [comment]
Egging On (Xbox Cloud/Remote Play):
![IMG_5625](https://github.com/user-attachments/assets/37a73eb1-c6d9-493e-8639-eedf1115dfcd)

**Comment by Kode-Z** [comment]
L.A. Noire (Remote Play):
![IMG_5704](https://github.com/user-attachments/assets/38a5a8c5-b8d0-435a-b46e-9ada310708a6)
![IMG_5692](https://github.com/user-attachments/assets/4bb46e84-3c7f-44c5-b0ff-c524a4a8f7e9)

**Comment by Kode-Z** [comment]
Silent Hill 2 (Xbox Cloud/Remote Play):
<img width="1398" height="645" alt="IMG_5747" src="https://github.com/user-attachments/assets/e4c180ff-990d-4016-9bba-be9e3f7a1f94" />

**Comment by Kode-Z** [comment]
Red Dead Redemption (Remote Play):
_Covers both the Xbox 360/Xbox Series X/S versions of the game!_

![IMG_5911](https://github.com/user-attachments/assets/4c0a0e32-d33b-4bf5-995f-f29c35399f9c)

**Comment by Kode-Z** [comment]
Resident Evil Requiem (Remote Play):
_For optimal play experience, change the sprint from “Toggle” to “Hold” in the controller menu!_

![IMG_7096](https://github.com/user-attachments/assets/10c373a4-d4c7-405e-ac73-43cf475f0767)
![IMG_6772](https://github.com/user-attachments/assets/f169f4e9-a703-46ed-8c49-590158c80920)

**Comment by Kode-Z** [comment]
Resident Evil 3 (Xbox Cloud/Remote Play):
![IMG_7116](https://github.com/user-attachments/assets/6b2ec4fd-007b-4611-8a05-8a1f50fde951)

Used Redphx’s RE2 layout as the template, made adjustments required for RE3.

**Comment by luisend2026** [comment]
can someone do a control for descenders plssssss

**Comment by tm070603** [comment]
Can someone do one for helldivers 2 please

**Comment by Darksoul2127** [comment]
Can u create custom touch controls for forza horizon 6 with the same layout as forza horizon 5, thank you.

**Comment by y7shwebb** [comment]
Can you create touch controls for  cod

**Comment by Kode-Z** [comment]
007 First Light (Xbox Cloud/Remote Play):
<img width="2796" height="1290" alt="IMG_7746" src="https://github.com/user-attachments/assets/6d350ca6-a957-46c5-a136-da2d8ea92f9b" />

**Reply by MurlZin** [reply]
Hello, Could you create a layout for the new Assassin's Creed Black Flag Resynced?, well, we recently received the release of this wonderful game, but something that frustrated me was the lack of touch screen controls. They could have at least used a base from what was in Odyssey. Well, please, if anyone can make a custom layout for this game, I would be extremely grateful, and I know many others would be too, because it's a really important game that deserves a custom layout.

**Reply by Kode-Z** [reply]
@MurlZin  are the controls for Odyssey the same as Black Flag Resynced?

**Reply by MurlZin** [reply]
> @MurlZin are the controls for Odyssey the same as Black Flag Resynced?

@Kode-Z  Well, no, but I think that, since they're from the same franchise, you could just use the layout and button positions from the Odyssey as a base for Assassin's Creed Black Flag Resynced, simply adapting the controls to their respective functions, because they have different controls, Besides the difference in the running mechanics between the two games, where in Odyssey we didn't have a run button, in Black Flag Resynced we have one, on the L button. Oh, anyway, I think it would be cool to do something like Resident Evil 9 Requiem compared to the layout of Resident Evil 2 Remake, 3 Remake, re7 and re4 remake, the games in the Resident Evil franchise in general

**Reply by Math848** [reply]
Hello can you create a layout for the game Enlisted?

**Reply by MurlZin** [reply]
@Kode-Z  I was just thinking, could you make a custom layout for Phasmophobia? It's simple and fun to play.

**Comment by martinezjaymark797-commits** [comment]
Basketball
